assembl/static2/js/app/components/login/assemblLogin.jsx

```diff
@@ -8,11 +8,15 @@ import { get, getFullPath, getContextual } from '../../utils/routeMap';
 class AsLogin extends React.Component {
   render() {
     const { slug } = this.props;
-    const next = this.props.next ? this.props.next : get('home', { slug: slug });
+    const next = this.props.next ?
+      this.props.next
+      : (
+        slug ? get('home', { slug: slug }) : null
+      );
     const errorMessage = ('error_message' in this.props && this.props.error_message) ? this.props.error_message : null;
     return (
       <div>
-        <form className="login" method="POST" action={getFullPath('ctxOldLogin', { slug: slug })}>
+        <form className="login" method="POST" action={slug ? getFullPath('ctxOldLogin', { slug: slug }) : getFullPath('oldLogin')}>
```

Comment:
Haha yeah. It was a very strange case of rebasing error or something. Anyways, merged your changes right back in there. On a merge, your changes should remain intact.